addons/hr_holidays/models/hr_accrual_plan.py

```diff
@@ -0,0 +1,90 @@
+# # -*- coding: utf-8 -*-
+
+import datetime
+import logging
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+class AccrualPlan(models.Model):
+    _name = "hr.accrual.plan"
+    _description = "Accrual Plan"
+    _order = 'sequence, id'
+
+    name = fields.Char('Accrual Plan', required=True)
+    company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True,default=lambda self: self.env.company)
```

Comment:
```suggestion
    company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.company)
```

Linting issue. You should install a linter like pylint to easily spot those issues.

Also the line could be split into 2.